examples/automm/object_detection/voc2coco.py

```diff
@@ -9,11 +9,6 @@
 5. TODO: remove invalid images?
 
 To use:
-1. run in root_dir:
-    grep -ERoh '<name>(.*)</name>' ./Annotations | sort | uniq | sed 's/<name>//g' | sed 's/<\/name>//g' > labels.txt
-2. run in root_dir/..:
-    ls Annotations/*.xml > pathlist.txt
-3. run here:
     python3 voc2coco.py --root_dir <root_dir> --train_ratio <train_ratio> --val_ratio <val_ratio>
```

Comment:
`python3 voc2coco.py --root_dir <root_dir>` to convert pre-existing split, or 
`python3 voc2coco.py --root_dir <root_dir> --train_ratio <train_ratio> --val_ratio <val_ratio>` to split and convert

Reply:
Addressed in 93229126d707afa87ea6656a4701d78b6d4833e0